packages/components/src/internal/query/selectRows.ts

```diff
@@ -22,6 +22,7 @@ export interface SelectRowsResponse {
     messages: Array<Record<string, string>>;
     queryInfo: QueryInfo;
     rowCount: number;
+    totalRows?: number;
```

Comment:
I'm not sure we should be making any changes beyond `includeTotalCount = false` to `selectRows.ts`. I think of this implementation as our ideal implementation where as `totalRows` in `selectRowsDeprecated` is a notion we're looking to get away from (sorry, realize I'm restating the same thing here I said in another comment).

Reply:
I see what you are saying. I went back and forth on this. I originally had removed totalRows, as mentioned. Then I was thinking that maybe it would be helpful as a way to know when using the response object that the query had actually requested the total row count, without having to know where that query was made in the cases where the response might be used / passed around. Should be easy enough to take this out and switch over the usages. I'll do that.

Reply:
Looks good.